web/packages/teleterm/src/ui/Search/pickers/results.story.tsx

```diff
@@ -476,6 +477,7 @@ const SearchResultItems = () => {
             <Component
               searchResult={searchResult}
               getOptionalClusterName={routing.parseClusterName}
+              isVnetSupported={true}
```

Comment:
I'm unsure about putting `AppItem` below, as `ExtraTopComponent`. It is not meant to be used there, also it should rather be put inside `<InteractiveItem>`.

Would be cool if we could show it as the regular search item, what would you say for this little hack?
```ts
isVnetSupported={
                searchResult.kind === 'app' &&
                !searchResult.resource.labels['no-vnet']
              }
```

Reply:
It's true that it's technically not meant to be used within `ExtraTopComponent`. But I'd rather keep the logic straightforward (and controlled directly through the props) rather than introduce magic labels – I did consider doing that too, but I mostly thought about special matching on resource name or something. A special label isn't that bad, but I think it's just harder to figure out than plain props.

Reply:
Yeah, the special label is a bit of magic, we can keep it as is.